Yoast/javascript
# P2-814 Implement appender according to design

## Summary


This PR can be summarized in the following changelog entry:

* [@yoast/schema-blocks] Fixes the styling of the block appender used in the `InnerBlocks` block instruction. 

## Relevant technical choices:

* 

## Test instructions

This PR can be tested by following these steps:

* Create a post.
* Add a job posting block.
* Select the job posting block (if it is not selected already).
* You should see a block appender at the bottom of the job posting.
	* It should look like [the block appender in the design](https://www.sketch.com/s/ab6ebb5c-a390-434c-bdf1-8e3240cadcef/a/e7aAGz).
	  <img width="597" alt="Screenshot 2021-04-13 at 11 22 38" src="https://user-images.githubusercontent.com/10195175/114529812-9f5b3600-9c4a-11eb-89ae-93dd032dc6fd.png">
	* **Note**: It may look slightly different based on the theme you have installed on your website. E.g. a different background and/or font. All in all the styling should look consistent with the rest of the theme. 
* Make sure that the block appender works as expected. 
	* E.g. remove the job description from the job posting and re-add it via the block appender.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  * The location and base salary inner blocks of the job posting, which have inner blocks to be able to add variations (e.g. remote location or office location), should **not** have an appender.

## UI changes

## Quality assurance


Fixes #
